subjects/devops/file-struct/README.md

```diff
@@ -0,0 +1,55 @@
+## File struct
+
+### Instructions
+
+Create the files and directories so that when you use the command `tree` below, the output will look like this:
+
+```console
+$ tree struct/
+struct/
+├── 0
+├── 1
+├── 2
+├── 3
+│   └── text.txt
+├── 4
+│   └── text2.txt
+├── 5
+├── 6
+├── 7
+├── 8
+├── 9
+└── A
+    └── text3.txt
+```
+
+Once it is done, use the command below to create the file `done.tar` to be submitted.
+
+```console
+$ tar -cf done.tar *
+$ ls
+0  1  2  3  4  5  6  7  8  9  A  done.tar
+```
+
+**Only `done.tar` should be submitted.**
+
+### Hints
```

Comment:
files creations hints is not exists,
you can give the student intro about touch usage

Reply:
it exists in the previous exercise so i don't think we need repetition but let me know if you want to add it again.